### ISSUE TITLE
Support showing images in 3d

### ISSUE THREAD

**Issue by K-Meech** [mobie/mobie-viewer-fiji#322]
The spec supports showing images in 3d (https://github.com/mobie/mobie.github.io/blob/master/schema/view.schema.json#L71), but this is not yet supported in the mobie viewer.

Only segmentations can currently be viewed in 3D.

**Comment by K-Meech** [comment]
I think these two fields: ```showImagesIn3d``` and ```resolution3dView``` are the last two things to be implemented in mobie for the bookmarks.
@tischi Would you have some time to look at showing images in 3d? I think this used to be supported in the old mobie - right?

**Comment by tischi** [comment]
@constantinpape 
Do we have a JSON field for showing images in 3D in the new spec already?

@K-Meech 
In the Java code it currently looks like this:
```
public abstract class AbstractSourceDisplay implements SourceDisplay
{
	// Serialization
	protected String name;
	protected double opacity = 1.0;
	protected boolean visible = true;
```

Where "visible" refers to visible in BDV...

Should we add another field?!

**Comment by tischi** [comment]
Ah, I just saw we have it here:

```
public class ImageSourceDisplay extends AbstractSourceDisplay
{
	// Serialization
	private List< String > sources;
	private String color;
	private double[] contrastLimits;
	private BlendingMode blendingMode;
	private boolean showImagesIn3d; // <---- THIS ONE
```

**Comment by tischi** [comment]
I am working on this [here](https://github.com/mobie/mobie-viewer-fiji/pull/534).

**Comment by constantinpape** [comment]
@tischi @K-Meech is this fixed by #534 now and can be closed?
